js/src/panes/pane.map.js

```diff
@@ -256,18 +256,19 @@ Wu.MapPane = Wu.Pane.extend({
     },
 
     _onMove : function () {
```

Comment:
Right, thanks for that. This is definitively wrong. I see now it was someone else who wrote it, that's why I'm confused. The whole function should be commented out. That's the immediate fix. Then probably there will be an issue with name not saving, but we'll solve that after. Can you comment it out now, first?